### ISSUE TITLE
Audio Section of v5 draft incorrectly pulled

### ISSUE THREAD

**Issue by rdoherty0** [Dash-Industry-Forum/DASH-IF-IOP#402]
The audio section of v5 should be pulled from the published "Audio Amendment for Guidelines.." which supersedes the text from v4.3.  It appears the section was pulled directly from v4.3 by mistake.

**Comment by mikedo** [comment]
Yes, that was done as a baseline to start from and is known to need work (see v5 meeting notes).   I don't think the amendment can be dropped in as is - it will need some merge work.  I had a v5 action item to take a stab at section 7, which includes video, audio and captions.  I started to incorporate some of the new audio guideline but it still needs a fair amount of work. I'll forward to you and Phil where I got on this.

**Comment by pmaness** [comment]
The correct material for section 7.4.6 is in the Audio Amendment section 3.9.4.6. You can probably take the entire section and cut and paste, then update the Table references and be done, though the Audio Amendment doesn't include the note about Rating. We had deemed it redundant as its covered elsewhere (verbatim in table 13).

**Comment by haudiobe** [comment]
We believe this is solved in https://dashif.org/guidelines/iop-v5/#part-8-audio.

**Comment by pmaness** [comment]
This issue was resolved long ago. It can be closed.
